### ISSUE TITLE
Broken Highlights On HEREDOCs with empty lines in them

### ISSUE THREAD

**Issue by zoffixznet** [MadcapJake/language-perl6fe#5]
See image:
![bug1](https://cloud.githubusercontent.com/assets/5747918/11793497/e8c2985e-a279-11e5-812a-aebb6b8bd37b.png)

**Comment by MadcapJake** [comment]
Would you post the text version of this?

**Comment by zoffixznet** [comment]
Here you go:

``` perl6
$dom = DOM::Parser.new(qq:to/END/);
<html>
  <head>
    <title>Real World!</title>
  <body>
    <ul>
      <li>
        Test
        <br>
        123
        <p>

      <li>
        Test
        <br>
        321
        <p>
      <li>
        Test
        3
        2
        1
        <p>
    </ul>
END
```

**Comment by MadcapJake** [comment]
Fixed in v1.2.4

**Comment by zoffixznet** [comment]
Sweet thanks :)
